backend/trading/trading_system.py

```
@@ -0,0 +1,531 @@
+                
+                for symbol in symbols:
+                    try:
+                        log = self.trading_iteration(symbol)
+                        
+                        if log and log.order:
+                            logger.info(
+                                f"Generated order: {log.order.side.value} "
+                                f"{log.order.quantity:.4f} {symbol} "
+                                f"(regime={log.regime.value})"
+                            )
+                    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)